.github/workflows/ci.yml

```diff
@@ -86,7 +86,14 @@ jobs:
           path: target/wasm32-wasi/release/
 
       - name: Test CLI
-        run: CARGO_PROFILE_RELEASE_LTO=off cargo test --package=javy-cli --release -- --nocapture
+        env:
+          CARGO_PROFILE_RELEASE_LTO: off
+        run: |
+          cargo build --package=javy-core --target=wasm32-wasi --release
+          cargo test --package=javy-cli --release -- --nocapture
+
+          cargo build --package=javy-core --target=wasm32-wasi --release --features=experimental_event_loop
+          cargo test --package=javy-cli --features=experimental_event_loop --release -- --nocapture
```

Comment:
This will result in CI taking materially longer to complete. Can we do the recompilation of `javy-core` and `javy-cli` with the `experimental_event_loop` feature as a separate GitHub Actions workflow? Similar to what we do for WPT. Or maybe rename the WPT pipeline and do a `cargo test` with the feature enabled and also perform a WPT run in that workflow? That way we continue to compile the `javy-core` module with the additional feature in parallel instead of serially.

Reply:
Oh right, yeah good point. I wouldn't use wpt, but a separate workflow seems reasonable.

Reply:
Fixed.